authorization-service/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "import-service",
+  "version": "1.0.0",
+  "description": "Serverless webpack example using Typescript",
+  "main": "handler.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "dependencies": {
+    "aws-lambda": "^1.0.6",
+    "aws-sdk-mock": "^5.1.0",
```

Comment:
do we need it in `dependencies `?

Also lambda under the hood has `aws-sdk`, so you should move `aws-lambda` to devDependencies